src/Components/Notes/NoteFooter.jsx

```diff
@@ -130,48 +133,60 @@ export default function NoteFooter({
        <TooltipIconButton
          title='Take Screenshot'
          size='small'
-         placement='bottom'
+         placement='top'
          onClick={() => {
            setScreenshotUri(viewer.takeScreenshot())
          }}
          icon={<PhotoCameraIcon className='icon-share'/>}
        />
       }
 
-      {editMode &&
-       <TooltipIconButton
-         title='Save'
-         placement='left'
-         icon={<CheckIcon className='icon-share'/>}
-         onClick={() => submitUpdate(repository, accessToken, id)}
-       />
-      }
-
       {isNote && !selected &&
        <TooltipIconButton
          title='Add Comment'
          size='small'
-         placement='bottom'
+         placement='top'
          selected={showCreateComment}
          onClick={selectCard}
          icon={<AddCommentOutlinedIcon className='icon-share'/>}
        />
       }
+      {editMode &&
+          <Box sx={{marginLeft: 'auto', padding: '0 0.5em'}}>
```

Comment:
Could the left-padding be moved to the parent to remove it from have it repeated in these 3 components?